DEVELOPMENT.md

```diff
@@ -40,13 +41,20 @@ source .venv/bin/activate # Activate and use your virtual environment
 # And once you're all done..
 deactivate
 ```
+### <a name='Forkrepo'></a>Fork the official repo
```

Comment:
I think these links likely generated with vscode extensions don't work with github. Would it be possible for you to correct the links in this document(not necessarily the result of your contributions) based on the format in README?

```markdown
* [Table of Contents](#table-of-contents)
* [Installation](#installation)
* [Gen AI Endpoint](#gen-ai-endpoint)
    * [Example](#example)
* [Examples](#examples)
    * [Async Example](#async-example)
    * [Synchronous Example](#synchronous-example)
* [Tips and Troubleshooting](#tips-and-troubleshooting)
```

Reply:
While updating the index for DEVELOP.md, I noticed the three last sections do not exist in this file:

```
* [Deployment Process](#DeploymentProcess)
	* [Automated Release Process](#AutomatedReleaseProcess)
	* [Manual Release Process](#ManualProcess)

```

Temporarily, I commented them. Do we have this info somewhere else?